bin/test-merges.js

```diff
@@ -7,7 +7,7 @@
 const Task = require("co-task");
 const process = require("process");
 const path = require("path");
-const GitHubApi = require("github");
+const GitHubApi = require("@octokit/rest");
 const shelljs = require("shelljs");
 const child_process = require("child_process");
 const github = new GitHubApi({debug: false});
```

Comment:
According to the relnotes for v13, support for "debug" was taken out then, so you can probably just remove the parameter from this line entirely.